ros_sam_msgs/package.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ros_sam_msgs</name>
+  <version>0.0.0</version>
+  <description>The ros_sam_msgs package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="aroefer@todo.todo">aroefer</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
missing license

Reply:
Good spot!

Reply:
it is getting a little sarcastic for tagging me on a missing license commit...